src/extra/components/adq.py

```diff
@@ -124,7 +124,7 @@ def __init__(self, data, channel, digitizer=None, pulses=None,
             raise ValueError('channel expected to be 2 or 3 characters, '
                              'e.g. 1A or 1_A')
 
-        self._channel_number = ord(self._channel_letter) - ord('A')
+        self._channel_number = ord(self._channel_letter) - ord('A') + 1
```

Comment:
Are we confident no-one is already using channel numbers as they are now, counting from zero?

Reply:
Yes, the numbers and letters are enumated at initialization time by the `AdqDigitizer` device and cannot be configured. They're not even (always) in the physical order in the crate.

Reply:
Sorry, perhaps I'm misunderstanding. At the moment, if I create `adq = AdqDigitizer(run,  '1C')`, then `adq.number` will give me 2 (A=0, B=1, C=2), yes? With this change, it becomes 3.  Are you confident that no-one has started using those numbers as they are?

(We can still change it even if someone might be using it - I just want to think about if we need to announce the change somehow)

Reply:
Apologies, I understand now. So the earlier behaviour was a bug, `adq.number` was always meant to represent the board _number_ in the sense it was used in `CONTROL` parameters (yes, someone funny used letters for fast data and numbers for slow data). As a matter of fact, `.channel_parameters` would've given you the wrong values as of now.

I highly doubt this parameter was used already directly, but if you feel strongly about it I can add it as a bugfix.

Reply:
OK, thanks. Let's mention it in the changelog, though since we install master every day it doesn't make much difference.